src/libpriv/rpmostree-scripts.cxx

```diff
@@ -234,10 +235,10 @@ lookup_script_action (const char *pkg_name,
                       const char *scriptdesc)
 {
   const char *pkg_script = glnx_strjoina (pkg_name, ".", scriptdesc+1);
-  const struct RpmOstreePackageScriptHandler *handler = rpmostree_script_gperf_lookup (pkg_script, strlen (pkg_script));
-  if (!handler)
+  if (rpmostreecxx::script_is_ignored (pkg_script))
```

Comment:
Note here how we're passing a C string directly into a function that accepts a Rust `&str`, no conversion dance necessary!  This is because [rust::Str](https://cxx.rs/binding/str.html) has a constructor that accepts a `const char*`.  It does currently validate UTF-8.